### PR TITLE
Remove Whiteboard

### PR DESCRIPTION
**Pull request by m273d15**
The current whiteboard is buggy and the HTML WB has the same
issues as the [HTML GUI](https://www.saros-project.org/contribute/html-gui.html).
We have no resources to maintain this approach.

Follow-up todos:
* remove corresponding GitHub label
* close corresponding issues

### CODE REVIEW

File: eclipse/plugin.xml [hunk 1]
```diff
@@ -1469,12 +1469,4 @@
          </adapter>
       </factory>
    </extension>
-   <extension
```
Comment (srossbach) [inline eclipse/plugin.xml]: This is not BUILD related.

Comment (m273d15) [reply eclipse/plugin.xml]: True, I will move the change into another commit.

**Comment by tobous** [comment]
As the whiteboard build files were previously covered by the gitignore, they are not automatically removed with this PR, meaning developers will still have to remove the `whiteboard` folder and its remaining contents themselves after checking out the newest changes (if they ever built the whiteboard locally before).

I don't know whether this is really necessary, but you could consider writing a short note on the mailing list or developer gitter after this is merged that some minor local cleanup is still necessary, just to avoid confusion.

**Comment by tobous** [comment]
You could consider adding an entry to the website pointing to the commit still containing the code in case somebody is interested in the whiteboard at some point in the future.

If I remember correctly, you also wanted to include the issues with the whiteboard with that. If you don't want to do that with this PR, you could just open an issue to remind us that the entry still has to be written.

**Pull request merged**